core/src/main/java/io/cucumber/core/options/CommandlineOptionsParser.java

```diff
@@ -65,66 +66,70 @@ private RuntimeOptionsBuilder parse(List<String> args) {
         while (!args.isEmpty()) {
             String arg = args.remove(0).trim();
 
-            if (arg.equals("--help") || arg.equals("-h")) {
+            if (arg.equals(CLIOptions.HELP) || arg.equals(CLIOptions.HELP_SHORT)) {
                 printUsage();
                 exitCode = 0;
                 return parsedOptions;
-            } else if (arg.equals("--version") || arg.equals("-v")) {
+            } else if (arg.equals(CLIOptions.VERSION) || arg.equals(CLIOptions.VERSION_SHORT)) {
```

Comment:
For readability static imports would be useful

Reply:
static import is good, I've just updated that